SJMaster/settings.py

```diff
@@ -129,6 +129,11 @@
 ]
 LOGIN_REDIRECT_URL = "/"
 LOGOUT_REDIRECT_URL = "/"
+STATICFILES_DIRS = [
```

Comment:
Please remove this as it's already in the file. (was added on #82 ).

Reply:
done